### PR TITLE
Add toolchains configs for a Windows run.

### PR DESCRIPTION
**Pull request by bansalvinayak**

### CODE REVIEW

File: configs/experimental/windows/0.1.0/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package(default_visibility = ["//visibility:public"])
```
Comment (nlopezgi) [inline configs/experimental/windows/0.1.0/BUILD.bazel]: Add license headers to all files (see example headers in files in our repo)

File: configs/experimental/windows/0.1.0/bazel_a320e4460f35767da7fc8eb824ce1f4b304b5e0f/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,148 @@
+package(default_visibility = ["//visibility:public"])
```
Comment (nlopezgi) [inline configs/experimental/windows/0.1.0/bazel_a320e4460f35767da7fc8eb824ce1f4b304b5e0f/BUILD.bazel]: why do these files have the .bazel suffix?

Comment (nlopezgi) [inline configs/experimental/windows/0.1.0/bazel_a320e4460f35767da7fc8eb824ce1f4b304b5e0f/BUILD.bazel]: what does the "a320e4460f35767da7fc8eb824ce1f4b304b5e0f" correspond to?
we'd rather this be bazel_<version> or something else humanly readable

Comment (bansalvinayak) [reply configs/experimental/windows/0.1.0/bazel_a320e4460f35767da7fc8eb824ce1f4b304b5e0f/BUILD.bazel]: Changed to bazel_experimental. Will figure out versioning story when we get to it.

**Pull request merged**